### PR TITLE
Remove max insurance fund before deleveraging

### PR DESCRIPTION
**Pull request by jayy04**
### Changelist
- Drop max insurance fund before deleveraging

### Test Plan
[Describe how this PR was tested (if applicable)]

### Author/Reviewer Checklist
- [ ] If this PR has changes that result in a different app state given the same prior state and transaction list, manually add the `state-breaking` label.
- [ ] If this PR isn't state-breaking but has changes that modify behavior in `PrepareProposal` or `ProcessProposal`, manually add the label `proposal-breaking`.
- [ ] If this PR is one of many that implement a specific feature, manually label them all `feature:[feature-name]`.
- [ ] If you wish to for mergify-bot to automatically create a PR to backport your change to a release branch, manually add the label `backport/[branch-name]`.
- [ ] Manually add any of the following labels: `refactor`, `chore`, `bug`.

### CODE REVIEW

**Review by lucas-dydx — APPROVED** [review]

File: protocol/x/clob/keeper/deleveraging.go [hunk 1]
```diff
@@ -160,8 +149,7 @@ func (k Keeper) CanDeleverageSubaccount(
 		return false, nil
 	}
 
-	// The insurance fund balance is less-than-or-equal to `MaxInsuranceFundQuantumsForDeleveraging`
-	// and the subaccount's total net collateral is negative, so deleveraging can be performed.
+	// The subaccount's total net collateral is negative, so deleveraging can be performed.
 	return true, nil
```
[the remark this answers is not in the document]
Comment (BrendanChou) [reply protocol/x/clob/keeper/deleveraging.go:153]: This is checked by the conditional on line 148

**Review by BrendanChou — APPROVED** [review]

File: protocol/x/clob/keeper/deleveraging.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (roy-dydx) [reply protocol/x/clob/keeper/deleveraging.go:153]: No it's not checked by the conditional on 148.

[the remark this answers is not in the document]
Comment (roy-dydx) [reply protocol/x/clob/keeper/deleveraging.go:153]: @coderabbitai what do you think?

[the remark this answers is not in the document]
Comment (roy-dydx) [reply protocol/x/clob/keeper/deleveraging.go:153]: Apologies coderabbit, I was just making sure you're not just yes-manning whenever you're corrected.

**Pull request merged**